templates/ovncontroller/bin/functions

```diff
@@ -49,34 +49,73 @@ function configure_external_ids {
     fi
 }
 
+# Returns the set difference between $1 and $2
+function set_difference {
+    echo "$(comm -23 <(sort <(echo $1 | xargs -n1)) <(sort <(echo $2 | xargs -n1)))"
+}
+
 # Configure bridge mappings and physical bridges
 function configure_physical_networks {
     local OvnBridgeMappings=""
+    local br_new=""
     for physicalNetwork in ${PhysicalNetworks}; do
         br_name="br-${physicalNetwork}"
-        ovs-vsctl --may-exist add-br ${br_name}
-        ovs-vsctl --may-exist add-port ${br_name} ${physicalNetwork}
         bridgeMapping="${physicalNetwork}:${br_name}"
         if [ -z "$OvnBridgeMappings"]; then
             OvnBridgeMappings=$bridgeMapping
+            br_new=$br_name
```

Comment:
(no action required) we could have a join() function that would receive an array and join the results. like:

```
joinWith() {
  local IFS="$1"
  shift
  echo "$*"
}

joinWith , bridgeMappings
joinWith ' ' bridges
```

we could have that used below to calculate br_current too...